src/Uno.UWP/Storage/Pickers/FileOpenPicker.Android.cs

```diff
@@ -19,6 +19,9 @@ public partial class FileOpenPicker
 		private static TaskCompletionSource<Intent?>? _currentFileOpenPickerRequest;
 
 		private const string StorageIdentifierFormatString = "Uno.FileOpenPicker.{0}";
+		private const string AnyWildcard = "*/*";
+		private const string ImageWildcard = "image/*";
+		private const string VideoWildcard = "image/*";
```

Comment:
Did you mean "video" ?

```suggestion
		private const string VideoWildcard = "video/*";
```